tests/exists_test.go

```diff
@@ -0,0 +1,33 @@
+package tests
+
+import (
+	"testing"
+	"time"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestExists(t *testing.T) {
+	conn := getLocalConnection()
+	testCases := []struct {
+		command  []string
+		expected []interface{}
+		delay    []time.Duration
+	}{
+		{
+			command:  []string{"SET key value", "EXISTS key", "EXISTS key2", "EXISTS key key2", "SET key2 value2", "EXISTS key key2", "EXISTS key key2 key3","SET key3 value ex 4", "EXISTS key3", "EXISTS key3 key2", "EXISTS key3 key2 key", "DEL key", "EXISTS key key2"},
```

Comment:
Let's break down the function into multiple lines for better readability.

Reply:
Sure. will improve the readability of this code.